Problem001/R/solution_1.R

```diff
@@ -0,0 +1,2 @@
+vector <- 1:1000
+sum(vector[vector%%3==0 | vector%%5==0])
```

Comment:
You can just put a EOL here? This will avoid some problems with tokenization and warnings in the feature when we enable statics linters for all languages. Thanks!